sceptre/cli/create.py

```diff
@@ -24,9 +30,12 @@ def create_command(ctx, path, change_set_name, yes):
     :type change_set_name: str
     :param yes: A flag to assume yes to all questions.
     :type yes: bool
+    :param disable_rollback: A flag to disable cloudformation rollback.
+    :type disable_rollback: bool
```

Comment:
When you're using type annotations, specifying the `:type` isn't necessary and is probably redundant.

Reply:
just following the existing pattern but can do as you suggest.

Reply:
We only started adding type annotations like a year and a half ago. Most of our existing patterns were around long before that. However, in the era of Python type annotations (py 3.5+, I believe), there's no longer any need to indicate types in docstrings. Sphinx understands type annotations just fine as well and interprets them the same way.

Reply:
done